gateway/webhook/webhook.go

```diff
@@ -244,6 +262,19 @@ func (webhook *HandleT) RequestHandler(w http.ResponseWriter, r *http.Request) {
 	ss.Report(webhook.stats)
 }
 
+func getWebhookFailureReason(err string, statusCode int) string {
```

Comment:
Can you rename the variable to `errMessage`? Because idiomatically speaking when someone sees an `err` variable in Go they know it's going to be an `error` but this isn't the case so for clarity we could simply rename it.